tests/unit/backend/layers/api/test_portal_api.py

```diff
@@ -611,40 +605,6 @@ def test__can_retrieve_created_collection(self):
         response = self.app.get(test_url.url, headers=no_cookie_headers)
         self.assertEqual("READ", json.loads(response.data)["access_type"])
 
-    # 🔴 used to be done by the database - we need to decide where this goes now
```

Comment:
This is still TODO, I would leave the test and add a strip method to the business logic.